torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -0,0 +1,61 @@
+#include <torch/csrc/jit/passes/normalize_ops.h>
+#include <c10/util/Exception.h>
+
+namespace torch {
+namespace jit {
+
+static const std::unordered_map<Symbol, Symbol> alias_map = {
```

Comment:
Sorry, my comment here didn't get saved, I guess. What about a comment here explaining how people should add (and test) new aliases?

Reply:
I added a comment. I would prefer not to put testing details in source code because it makes our code more verbose and the details would likely grow stale. If someone wants to look at how something is tested the typical thing is to look at git blame or ask around.

Reply:
Sounds good.